src/hb-subset.h

```diff
@@ -145,6 +154,25 @@ hb_subset_input_set_flags (hb_subset_input_t *input,
 HB_EXTERN hb_face_t *
 hb_subset_or_fail (hb_face_t *source, const hb_subset_input_t *input);
 
+HB_EXTERN hb_face_t *
+hb_subset_from_plan_or_fail (hb_subset_plan_t *plan);
+
+HB_EXTERN hb_subset_plan_t *
+hb_subset_plan_create (hb_face_t           *face,
```

Comment:
Probably also `_create_or_fail` instead of `_create`.

Reply:
plan_create always returns a valid pointer. It uses an in_error() to check for failure.

Reply:
Would it be useful for our purposes though?

Reply:
Yeah I could see that (nullptr) being more intuitive for end users. I'll change create to be or_fail.